test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -56,6 +56,12 @@ class HttpConnectionManagerConfigTest : public testing::Test {
   NiceMock<Tracing::MockHttpTracerManager> http_tracer_manager_;
   std::shared_ptr<NiceMock<Tracing::MockHttpTracer>> http_tracer_{
       std::make_shared<NiceMock<Tracing::MockHttpTracer>>()};
+  void httpConnectionManagerConfigHelper(
```

Comment:
Can we have createHttpConnectionConfig with a yaml string as an input?

Reply:
@dio I will do that but what about  
`HttpConnectionManagerConfig config(parseHttpConnectionManagerFromV2Yaml(yaml_string), context_,
                                     date_provider_, route_config_provider_manager_,
                                     scoped_routes_config_provider_manager_, http_tracer_manager_);`
?